src/back/controllers/api/routes/profile/myProfile.controller.spec.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const MyProfileController = require('./myProfile.controller');
+const ProfileController = require('./profile.controller');
+const profileProperties = require('./profileProperties');
+
+describe('MyProfileController', () => {
+    let sut;
+
+    beforeEach(() => {
+        sut = new MyProfileController();
+    });
+
+    it('should extend profile controller', () => {
+        sut.should.be.instanceOf(ProfileController);
```

Comment:
I want to test that all functionality of ProfileController is available in MyProfileController